src/components/markdown/custom-components.tsx

```diff
@@ -118,6 +151,12 @@ const customSatellytesComponents = {
     return <DetailsStyled {...props}>{props.children}</DetailsStyled>;
   },
   a(props) {
+    if (props.href && props.href?.startsWith('#')) {
+      return (
+        <AnchoredLinkStyled {...props}>{props.children}</AnchoredLinkStyled>
+      );
+    }
```

Comment:
before this condition was buried in styled components now it's a a much clearer place 🙏